post.md

```diff
@@ -163,4 +164,12 @@ const findErrors = (fieldName: string, errors: ZodIssue[]) => {
 };
 ```
 
-In conclusion, Zod provides a powerful and flexible way to handle form validation in a Next.js application. By combining it with server-side actions and the `useFormState` hook, we can create robust forms with clear and specific error messages.
+## Wrapping it Up
+
+Zod provides a powerful and flexible way to handle form validation in a Next.js application. We can apply server-side validation to our forms and still present a great user experience by following a few simple steps:
+
+- Define a Zod schema for each server action
+- Validate user input in your server action using the Zod schema
+- Return an array of errors if validation fails
+- In your client component, use the `useFormState` hook to receive validation errors
+- Display the validation errors to the user
```

Comment:
Much better conclusion, but something still feels a bit off... Maybe try something like this?

> Zod provides a powerful and flexible way to handle form validation in Next.js applications. Using Zod, we can implement server-side validation to our forms and still present a great user experience by following a few simple steps: _[insert steps here]_